src/csexp.mli

```diff
@@ -367,3 +367,7 @@ module Make (Sexp : Sexp) : sig
   end
   [@@deprecated "Use Parser module instead"]
 end
+
+type t =
+  | Atom of string
+  | List of t list
```

Comment:
It's a bit odd to have this type on its own. We should also do `include S with type sexp := t`. This does require introducing `S`, but it seems worth it to me.

Reply:
Sure, I did it.